core/src/reporter/data-feed.ts

```diff
@@ -26,6 +27,7 @@ export async function buildReporter(redisClient: RedisClientType, logger: Logger
     reporterQueueName: REPORTER_AGGREGATOR_QUEUE_NAME,
     concurrency: DATA_FEED_REPORTER_CONCURRENCY,
     delegatedFee: [BAOBAB_CHAIN_ID, CYPRESS_CHAIN_ID].includes(chainId) ? true : false,
-    _logger: logger
+    _logger: logger,
+    _worker: dataFeedReporter
```

Comment:
I think we have to pass this to VRF and RR reporter as well, no?

Reply:
default value for `_worker` is set as `reporter`. you can check it from reporter factory


```
export async function factory({
  redisClient,
  stateName,
  service,
  reporterQueueName,
  concurrency,
  delegatedFee,
  _logger,
  providerUrl = PROVIDER_URL,
  chain = CHAIN,
  _worker = reporter
}: {
  redisClient: RedisClientType
  stateName: string
	@@ -28,6 +30,7 @@ export async function factory({
  providerUrl?: string
  chain?: string
  _logger: Logger
  _worker?: workerType
})
```